### DISCUSSION TITLE
How can I detect twill within ComposerServiceProvider.php ?

### DISCUSSION THREAD

**Discussion by adam-jones-net** [area17/twill#1532] [Q&A]
Within my apps main ComposerServiceProvider.php  file I want to add some logic to detect if the page being generated is a page within Twill, so that I can then add a specific extra require_once() include I want to add.

Sure I could check the url path or something but I wondered if there is something a lot better, for example a way of using a specific View::composer syntax or a function that can clarify if Twill is running ?

**Comment by haringsrob** [comment]
Hey @arkid,

As Twill is headless, it has no idea about your front-end.

So I guess is that your suggestion of resolving the path might be your best bet here.
